packages/lab/src/combo-box-next/useComboBox.tsx

```diff
@@ -64,25 +63,23 @@ export const useComboBox = ({
   }, [selectedItem]);
 
   const focusHandler = (event: FocusEvent<HTMLInputElement>) => {
-    setOpen(true);
     listFocusHandler(event);
     onFocus?.(event);
   };
 
-  const blurHandler = (event: FocusEvent<HTMLInputElement>) => {
```

Comment:
check keyboard nav, you might still need blur

Reply:
In what respect, enter and esc are handled by the keyDownHandler or are you referring to a user being able to tab onto the next component when the combobox is open ?

Reply:
yeah, tab in keyboardhandler might do

Reply:
Added blur event for keyboard nav